### PR TITLE
Add test cases for valkey-benchmark negative path

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
Added the test case to check the wrong host case for valkey-benchmark.

### CODE REVIEW

File: tests/integration/valkey-benchmark.tcl [hunk 1]
```diff
@@ -37,6 +37,20 @@ tags {"benchmark network external:skip logreqres:skip"} {
             default_set_get_checks
         }
 
+        test {benchmark: negative path} {
+            set wrong_host 127.0.0.0
```
Comment (xbasel) [inline tests/integration/valkey-benchmark.tcl:41]: Hostnames/IPs and ports seem pretty basic; was there a specific reason to test these, or was it due to an edge case encountered or a broader fix?

Comment (Shivshankar-Reddy) [reply tests/integration/valkey-benchmark.tcl:41]: Hi @xbasel, Yes you are right, this is actually motivated by this PR https://github.com/valkey-io/valkey/pull/1029.

File: tests/integration/valkey-benchmark.tcl [hunk 2]
```diff
@@ -37,6 +37,20 @@ tags {"benchmark network external:skip logreqres:skip"} {
...
+            set wrong_port 6500
+            foreach e {valkeybenchmark valkeybenchmarkuri valkeybenchmarkuriuserpass} {
+                if {$e eq {valkeybenchmarkuriuserpass}} {
+                    set cmd [valkeybenchmarkuriuserpass $wrong_host $wrong_port "default" pass "-c 5 -n 10 -t set"]
```
Comment (ranshid) [inline tests/integration/valkey-benchmark.tcl:45]: no nead for set test and multiple clients IMO. just iuse a simple -I

File: tests/integration/valkey-benchmark.tcl [hunk 3]
```diff
@@ -37,6 +37,20 @@ tags {"benchmark network external:skip logreqres:skip"} {
             default_set_get_checks
         }
 
+        test {benchmark: negative path} {
```
Comment (ranshid) [inline tests/integration/valkey-benchmark.tcl:40]: negative_path suggested a very high level test.  I am just not sure why we decided to focus on this negative test while there are potentially many more negative testing we can add (eg bad arguments, TLS authentication, cluster discovery etc...). 
Anyway I would suggest to change the name to more explicitly reflect the test.
```suggestion
        test {benchmark: unable to connect to bad host} {
```

**Review by ranshid — COMMENTED** [review]
I think there is no big blocker here. Only the motivation is a little bit vague IMO.
For example why did we decide to focus only on this scenario?
